synapse/app/homeserver.py

```diff
@@ -88,11 +86,10 @@ def build_resource_for_web_client(hs):
                 "the location of the source to serve via the configuration\n"
                 "option `web_client_location`\n\n"
                 "To install the `matrix-angular-sdk` via pip, run:\n\n"
-                "    pip install '%(dep)s'\n"
+                "    pip install matrix-angular-sdk\n"
                 "\n"
                 "You can also disable hosting of the webclient via the\n"
                 "configuration option `web_client`\n"
-                % {"dep": DEPENDENCY_LINKS["matrix-angular-sdk"]}
```

Comment:
String substitution could be failing because there is no key "matrix-angular-sdk" in DEPENDENCY_LINKS, see https://github.com/matrix-org/synapse/blob/master/synapse/python_dependencies.py#L76
Also it seems to be put into CONDITIONAL_REQUIREMENTS as "web_client".
  
I'm sort of new here so I could be wrong. 😛